app/classifier/classifier.jsx

```diff
@@ -262,35 +262,42 @@ class Classifier extends React.Component {
   }
 
   completeClassification(e) {
+    const { classification, onComplete, interventions, project, subject, user, workflow } = this.props;
     const originalElement = e.currentTarget;
     const isCmdClick = e.metaKey;
+    const annotations = this.state.annotations.slice();
+    let workflowHistory = this.state.workflowHistory.slice();
+    const taskKey = workflowHistory[workflowHistory.length - 1];
+
+    const showIntervention = user &&
+      user.intervention_notifications &&
+      (interventions.notifications.length > 0);
+    const showSummary = !workflow.configuration.hide_classification_summaries ||
+      this.subjectIsGravitySpyGoldStandard();
+    const showLastStep = showIntervention || showSummary;
+
     // don't swallow cmd-click on links
     if (!isCmdClick) {
       e.preventDefault();
     }
-    this.props.classification.update({
+    classification.update({
       annotations: this.processAnnotations(this.state.annotations.slice()),
       'metadata.session': getSessionID(),
       'metadata.finished_at': (new Date()).toISOString(),
       'metadata.viewport': {
         width: innerWidth,
         height: innerHeight
+      },
+      'metadata.interventions': {
+        message: showIntervention,
```

Comment:
Do we want message to always be a boolean here? If there isn't a user present I believe it would be set to `null`

Reply:
Good catch. I’ll update this and add some tests for the classification metadata.

Reply:
There's a test for this now, but the bug itself isn't fixed yet.